signalfx-tracing/signalfx-nodejs-tracing/mysql/server.js

```diff
@@ -0,0 +1,36 @@
+// The core of the auto-instrumentation process is made in the
+// tracer.js module.  It consists of a single import and invocation
+// of the 'signalfx-tracing' module.
+const tracer = require('./deedScheduler/tracer');
+
+const Koa = require('koa');
+const logger = require('koa-logger');
+const scheduler = require('./deedScheduler');
+const server = new Koa();
```

Comment:
do you mind having the require statements separate from server and port initializations?